config/test.js

```diff
@@ -15,6 +15,9 @@ module.exports = {
     http: 0,
   },
   behindForwarding: false,
+  adapterManager: {
+    path: './adapters-test',
+  },
```

Comment:
You should probably also define the API level constant as well.

Reply:
So does this mean that you also think that the paths in config/default.js should also be constants? Since this is a config file, this is the source of the constant.

Reply:
No, I was just suggesting that you define it here, so that the API level checks are actually tested.

Reply:
Sorry - I misread your comment. The way that the config files work, the contents of test.js is overlaid with default.js. So test.js only provides things which need to be different from default.js.

See: https://github.com/lorenwest/node-config/wiki/Configuration-Files

In our case, default.js corresponds to default.EXT and test.js corresponds to {deployment}.EXT

Reply:
Ah, I see. In that case, carry on.